sbnanaobj/StandardRecord/SRHit.cxx

```diff
@@ -0,0 +1,38 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRHit.cxx
+// \brief   An SRHit is a high level shower object.  It knows its
+//          direction and length, but does not own its cell hits.
+////////////////////////////////////////////////////////////////////////
+
+#include "sbnanaobj/StandardRecord/SRHit.h"
+
+#include <climits>
+
+namespace caf
+{
+  SRSpacePoint::SRSpacePoint():
```

Comment:
If I understood correctly from my Pull Request with the `SRNuID` object, the ideal was going to be moving towards putting initial values in the header file directly rather than in the constructor.

See `StandardRecord/SRNuID.{cxx,h}`